inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/split.cpp

```diff
@@ -27,8 +27,27 @@ INSTANTIATE_TEST_CASE_P(smoke_NumSplitsCheck, SplitLayerTest,
                                 ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
                                 ::testing::Values(InferenceEngine::Layout::ANY),
                                 ::testing::Values(InferenceEngine::Layout::ANY),
-                                ::testing::Values(std::vector<size_t >({30, 30, 30, 30})),
+                                ::testing::Values(std::vector<size_t>({30, 30, 30, 30})),
                                 ::testing::Values(CommonTestUtils::DEVICE_MYRIAD)),
                         SplitLayerTest::getTestCaseName);
 
+INSTANTIATE_TEST_CASE_P(smoke_SplitWithDiffOutsCheck, splitWithUnusedOutputsTest,
```

Comment:
Nitpick: just for consistency change the name of the test case `smoke_SplitWithDiffOutsCheck`

Reply:
Changed.